ui-design-lib/src/main/java/co/electriccoin/zcash/ui/design/component/Text.kt

```diff
@@ -46,8 +46,8 @@ fun Body(
 @Composable
 fun Small(
     text: String,
-    modifier: Modifier = Modifier,
-    textAlign: TextAlign
+    textAlign: TextAlign,
+    modifier: Modifier = Modifier
```

Comment:
It’s not to have the modifier type last, but to have all parameters with default arguments last.  This works better from the caller’s perspective because it makes the call simpler.